proof/encoding.go

```diff
@@ -480,10 +479,6 @@ func GroupKeyRevealEncoder(w io.Writer, val any, buf *[8]byte) error {
 }
 
 func GroupKeyRevealDecoder(r io.Reader, val any, buf *[8]byte, l uint64) error {
-	if l > btcec.PubKeyBytesLenCompressed+sha256.Size {
```

Comment:
Older nodes will have to update, they won't be able to validate the new group key reveals. However, updated nodes will still be able to handle the older group key reveal versions.

Reply:
What I mean is that won't older nodes reject the new proofs all together due to a parsing failure? Which will cause temporary Universe state divergence.

Reply:
Yes, the older nodes won't be able to correctly parse the new group key reveal. But they should receive an error message rather than decode incorrectly. Older nodes will have to upgrade to handle newly minted assets. And so will universe servers.